src/date-range-picker/index.tsx

```diff
@@ -109,7 +110,8 @@ const DateRangePicker = React.forwardRef(
     const { __internalRootRef } = useBaseComponent('DateRangePicker');
     checkControlled('DateRangePicker', 'value', value, 'onChange', onChange);
 
-    value = isDateOnly(value) ? value : shiftTimeOffset(value, timeOffset);
+    const normalizedTimeOffset = normalizeTimeOffset(value, getTimeOffset, timeOffset);
+    value = isDateOnly(value) ? value : shiftTimeOffset(value, normalizeTimeOffset(value, getTimeOffset, timeOffset));
```

Comment:
[minor] seems to be some duplication in these two lines

Reply:
oh, indeed 👍